README.md

```diff
@@ -1,57 +1,116 @@
 
 ## Commands
 
-The project uses Make commands listed in the [Makefile](Makefile) for ease of development. The commands are as follows:
+The project uses Make commands listed in the [Makefile](Makefile) for ease of development. The available commands are as follows.
+
+| Command                 | Target    | Description
+|-------------------------|-----------|----------------------------------------------------
+| make build              | Local     | (Re-)Builds local images listed in the docker-compose
+| make run                | Local     | Launches the application using Docker (builds the images if required)
+| make restart            | Local     | Stops the app, rebuilds the images, and restarts the app
+| make close              | Local     | Stops and removes any locally running containers
+| make clean              | Local     | Purges containers, images, and volumes
+| make client-test        | Local     | Runs client unit tests
+| make create-nsp         | OpenShift | Creates the requisite Network Security Policies in OpenShift
+| make create-database    | OpenShift | Creates the database service in OpenShift
+| make create-server      | OpenShift | Creates the server service in OpenShift
+| make create-client      | OpenShift | Creates the client servvice in OpenShift
+| make oc-all-clean       | OpenShift | Removes all OpenShift objects associated with the application (does not affect persisted objects)
+| make oc-database-clean  | OpenShift | Removes all OpenShift objects associated with the database service (does not affect persisted objects)
+| make oc-persisted-clean | OpenShift | Removes all persisted objects associated with the application (NSPs, volume claims, and secrets)
+| make oc-server-clean    | OpenShift | Removes all OpenShift objects associated with the server service
+| make oc-client-clean    | OpenShift | Removes all OpenShift objects associated with the client service
+
+Refer to the [Makefile](Makefile) for arguments required for the above commands.
+
+## Using Windows
+
+We can run make commands in Windows using the following steps.
+- Install [GNUWin](http://gnuwin32.sourceforge.net/)
+- Find the installation location of GNUWin and its bin folder (eg: `C:\\Program Files (x86)\\GnuWin32\\bin`)
+- Open the Environment Variables sections of Windows System Properties (see [instructions](https://docs.oracle.com/en/database/oracle/r-enterprise/1.5.1/oread/creating-and-modifying-environment-variables-on-windows.html#GUID-DD6F9982-60D5-48F6-8270-A27EC53807D0))
+- Edit the `Path` environment variable and add the GNUWin bin directory from above (note that paths are comma separated)
+- Reopen PowerShell or CMD and run Makefile commands
+
+## Database Migrations
```

Comment:
@ashtonmeuser Suggest moving Database Migrations below Getting Started since it is an important functionality implemented in docker and Openshift.